commands/Discord/help.js

```diff
@@ -0,0 +1,63 @@
+const Discord = require('discord.js');
+
+module.exports = {
+  name: 'help',
+  description:
+      'Shows help for a specific command, or lists all available commands.',
+  usage: '[command]',
```

Comment:
including command name as well  is better right?

Reply:
If we do that, then the name is hard-coded. Instead, we can have it filled dynamically, so if we use a command's alias along with the help command, the usage will show the alias instead of the actual command name.